.github/workflows/_netlify.yml

```diff
@@ -10,26 +10,33 @@ on:
       
 jobs:
   deploy-to-netlify:
-    name: Deploy ${{ input.service }} on Netlify
+    name: Deploy ${{ inputs.service }} on Netlify
     runs-on: ubuntu-latest
     env:
       DOCKER_BUILDKIT: 1
       BUILDKIT_PROGRESS: plain
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: Check for changes in ${{ inputs.service }}
         run: |
           changed=$(scripts/monorepo.sh ${{ inputs.service }} ${{ github.ref_name }})
           echo "::set-output name=changed::$changed"
         shell: bash
         id: check_changes
       - name: Check if forked PR
+        id: is_forked
         run: |
-          echo "::set-output name=is_forked::$(${{github.repository_owner != 'Unlock Protocol' && github.event.pull_request.merged == true}})
+          echo "::set-output name=is_forked::${{github.repository_owner != 'Unlock Protocol' && github.event.pull_request.merged == true}}"
       - name: Set target environment
+        id: set_target
         run: |
-          echo "::set-output name=target::$(${{ contains(github.base_ref, 'production') || input.service == 'unlock-protocol-com' }} && 'prod' || 'staging')"
+          echo "::set-output name=target::${{ contains(github.base_ref, 'production') && 'prod' || 'staging' }}"
+      - name: Show Netlify script params
```

Comment:
debugging?

Reply:
not sure why we were using this `|| input.service == 'unlock-protocol-com'` condition anymore... This may lead to deploy the website from `master` (without using the `production` branch). Is that intended?

Reply:
yes! Sorry for the delay, we did not have a "staging" for unlock-protocol.com. master was deploying to the prod site.